extensions/data-workspace/src/test/workspaceService.test.ts

```diff
@@ -323,4 +323,38 @@ suite('WorkspaceService', function (): void {
 		should.strictEqual(updateWorkspaceFoldersStub.calledOnce, true, 'updateWorkspaceFolders should have been called');
 		onWorkspaceProjectsChangedDisposable.dispose();
 	});
+
+	test('createProject passes all property', async () => {
+		this.timeout(30000);
+		// Create a new instance of WorkspaceService
+		const service = new WorkspaceService();
+
+		// Stub the createProject method so we can verify how it's called
+		const createProjectStub = sinon.stub(service, 'createProject').resolves();
+
+		// Define test parameters
+		const name = 'TestProject';
+		const location = vscode.Uri.file('/tmp/TestProject');
+		const projectTypeId = 'sqlproj';
+		const projectTargetVersion = 'SqlAzureV12';
+		const sdkStyleProject = true;
+		const configureDefaultBuild = true;
+
+		// Call the method under test with all parameters, including configureDefaultBuild
+		await service.createProject(name, location, projectTypeId, projectTargetVersion, sdkStyleProject, configureDefaultBuild);
```

Comment:
What scenario does this test? You stub out `createProject` then checks that it has been called with the same parameters that you passed in?

Reply:
I think a better setup would be to mock the vscode quickpick, then verify that `createProject` is called with the right parameters